msal/src/main/java/com/microsoft/identity/client/SingleAccountPublicClientApplication.java

```diff
@@ -224,21 +184,6 @@ private void getCurrentAccountFromSharedDevice(@NonNull final CurrentAccountCall
         final String methodName = ":getCurrentAccountFromSharedDevice";
 
         //TODO: migrate to Command.
-        try {
```

Comment:
broker checks no longer needed ? are we doing it before?

Reply:
I moved the check to getBrokerAccountMode instead, meaning that we can safely assume that whenever isSharedDevice = true, it can talk to broker.

If it can't talk to broker, the flag will always be set to false.